mq/3/reference/dlq/index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: IronMQ Dead Letter Queue Reference
+layout: default
+section: mq-v3
+breadcrumbs:
+  - ['Reference', '/reference']
+  - ['Dead Letter Queue', '/dlq']
+---
+
+<section id="toc">
+  <h3>Table of Contents</h3>
+  <ul>
+    <li><a href="#overview">Overview</a></li>
+    <li><a href="#creating">Creating a Dead Letter Queue</a></li>
+    <li><a href="#deleting">Deleting a Dead Letter Queue</a></li>
+    <li><a href="#behavior">Behavior</a></li>
+    <li><a href="#use-case">One Time Delivery Use Case</a></li>
+  </ul>
+</section>
+
+<h2 id="overview">Overview</h2>
+
+Dead Letter Queues are queues that can be used to send messages to that for
```

Comment:
Awkward wording in this paragraph, especially the first sentence. My attempt:

A source queue may assign a dead letter queue, which is used as a destination for messages that could not otherwise be processed for some reason. Messages that have timed out a certain number of times on the source queue are placed onto the dead letter queue. Messages on the dead letter queue can then be processed again or analyzed to try to determine why they weren't successfully processed in the first place, without having to throw them away.